rc/filetype/ninja.kak

```diff
@@ -40,41 +40,29 @@ add-highlighter shared/ninja regions
 # `#`
 add-highlighter shared/ninja/comment region '#' '\n' fill comment
 
-# `rule`
-add-highlighter shared/ninja/rule region '^rule' '\n' group
-add-highlighter shared/ninja/rule/rule regex '^rule' 0:keyword
-
-# `command`
-add-highlighter shared/ninja/command region '^\h+command' '[^$]\n' group
-add-highlighter shared/ninja/command/command regex 'command' 0:variable
-add-highlighter shared/ninja/command/equal regex '=' 0:operator
-add-highlighter shared/ninja/command/linebreak regex '\$$' 0:operator
-add-highlighter shared/ninja/command/variables regex '\$\w+|\$\{\w+\}' 0:value
+# `subninja`, `include`, `rule`, `pool` and `default` declarations
+add-highlighter shared/ninja/sirpd region '^(subninja|include|rule|pool|default)\b' '[^$]\n' group
+add-highlighter shared/ninja/sirpd/default regex '^(subninja|include)\b' 0:module
+add-highlighter shared/ninja/sirpd/rulepool regex '^(rule|pool|default)\b' 0:keyword
+add-highlighter shared/ninja/sirpd/linebreak regex '\$$' 0:operator
```

Comment:
Those lines could be factorised into one, and remain fairly readable.

Reply:
I’m afraid I’m out of time for now, feel free to take over or suggest and I will validate.